wiki/Tournaments/POT/2022/en.md

```diff
@@ -0,0 +1,393 @@
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper | [hamdanpapeng](https://osu.ppy.sh/users/10854143) ![][flag_ID] | 1 | **5** | ![][flag_ID] **[nasje](https://osu.ppy.sh/users/7579498)** | [#1](https://osu.ppy.sh/community/matches/98501050) |
+
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+2. The mappools for each round will be announced by the Tournament Management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the Tournament Management. If there are any player(s) who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the `#scheduling` channel in the tournament's Discord server.
+4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+5. If a player does not show up within **10 minutes** of the start time, their opponent wins by default.
+6. If no staff or referee is available, the match will be postponed.
+7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards players who perform better in general during the course of the beatmap regardless of their remaining health at the end.
+8. If a player disconnects, it will be treated as if they had failed the beatmap. 
+   - A match can be rematched for disconnects that occur within a few seconds after the beatmap has been started by the referee.
+9. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes at most. 
+   - In case the disconnected player fails to report back after the maximum allowance of 15 minutes had passed, the opposing player may be declared to win the match by default.
+10. Lag is not a valid reason to nullify a beatmap.
+11. If any problems during the match occur, the Tournament Management will make a decision based on the referee's report.
+12. It is expected that all players be polite and respectful to each other. Penalties will be given upon violation.
+    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from the future iterations of the tournament by the Tournament Management.
+    - Usage of any tools and programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+
+### Tournament registration
+
+1. All interested players are required to register into the tournament individually.
+   - In order to be eligible to play in the tournament, a player must be able to attest their status as a registered student at (or as an alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) by showing a valid [EMAS UI credential](http://emas.ui.ac.id) to the Tournament Management.
```

Comment:
```suggestion
   - In order to be eligible to play in the tournament, a player must be able to attest their status as a registered student at (or alumni of) [Universitas Indonesia's Faculty of Computer Science](https://cs.ui.id) by showing a valid [EMAS UI credential](http://emas.ui.ac.id) to the Tournament Management.
```

Reply:
hmm i'll leave this one untouched to keep things consistent with what's written in the opening paragraph (Line 11) i think...

Reply:
well i suggested the same thing on the 2021 article and you applied that one sooo....

Reply:
oh D:

okok will apply this